### DISCUSSION TITLE
Workflows won't run in a forked repository

### DISCUSSION THREAD

**Discussion by rgov** [community/community#29029] [Actions] [answered]
I forked a repository which already had some GitHub workflows configured. However, when I make new commits to my fork, the workflows aren't running, and aren't even detected in my repository; the Workflows list is empty: https://github.com/rgov/super/actions

If it's relevant, some of my commits were to [this workflow file](https://github.com/rgov/super/blob/rzg/github-ci-test/.github/workflows/docker-image.yml) itself.

I checked under the Actions tab and under Settings > Actions and do not see anything that would obviously disable them. "Actions permissions" are set to "allow all."

**Comment by rgov** [comment, marked as answer]
Ah, the workflows are only configured to run on the `main` branch. GitHub apparently only shows workflows that have executed at least once.

**Comment by Andromelus** [comment]
Hey.
I also forked a [repo](https://github.com/Andromelus/hdfs/pull/1) would like to do some stuff with the already defined workflow. But I don't understand how to(or even if I can) trigger the worflow, at least manually.

Do you have some sources/insight on that ?

**Comment by FloStar3000** [comment]
In case someone runs into the same issue as me:
1st finding: `master` counts as `main`
2nd finding: There was no workflow file recognized in the "Actions" tab. I wanted to run the `.github/workflows/ci.yml` file that was on the `master`branch on the main repo. I just created an empty `.github/workflows/main.yml` which then led the `ci.yml`to be recognized
